README.md

```diff
@@ -6,6 +6,40 @@ Repo currently focuses on following along with the beam "getting started"
 materials: https://beam.apache.org/get-started/
 
 
+## System check
+
+To start, run the built-in copy of the word-count example with the following command,
+just to make sure that Apache Beam is correctly installed.
+
+```bash
+python -m apache_beam.examples.wordcount_minimal \
+  --input data/words.txt \
+  --output data/wordcounts_official_example.txt
+```
+
+This outputs a file `wordcounts_official_example.txt-00000-of-00001`. Why doesn't it
+match the requested output file name?
```

Comment:
When we find the answer to these questions let's update the doc to explain it :)